starter/src/main/resources/application.properties

```diff
@@ -25,3 +25,6 @@ spring.kafka.consumer.value-deserializer= org.apache.kafka.common.serialization.
 spring.kafka.consumer.properties.spring.json.trusted.packages=*
 
 
+spring.security.oauth2.client.registration.facebook.client-id=579437707268071
```

Comment:
I think we should not have these in the public repository... 
Probably we could store this in environment variables?

Reply:
Totally agree. Going to fix